src/entities/booking/BookingCurrentItem/typing.ts

```diff
@@ -0,0 +1,6 @@
+import { type TBookingItem } from '@/shared/ui/BookingItem';
+
+export type TBookingCurrentItem = Pick<TBookingItem, 'photo' | 'name'> & {
+  dates: string;
+  comment: string;
+};
```

Comment:
А если так?
```suggestion
export type TBookingCurrentItem = Required<Omit<TBookingItem, 'category' | 'button'>>
```
Чтобы не прописывать dates и comment повторно, а исключить ненужное из типа базового компонента и оставшиеся свойства сделать обязательными

Reply:
Отлично, так сделаю. Не знал про Required<>. Но я хотел бы оставить Pick, чтобы в случае появления новых полей не делать лишних действий.